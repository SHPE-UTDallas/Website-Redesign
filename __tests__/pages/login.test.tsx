import { shallow} from 'enzyme';
import Login from '../../pages/login'
import toJson from "enzyme-to-json";

it('renders correctly', () => {
  const wrapper = shallow(<Login />);
  expect(toJson(wrapper)).toMatchSnapshot();
});