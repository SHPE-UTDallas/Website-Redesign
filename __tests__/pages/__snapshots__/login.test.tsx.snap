// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Layout
  title="SHPE UTD"
>
  <div
    className="container"
  >
    <section
      className="section mt-4"
    >
      <div
        className="columns is-centered is-vcentered"
      >
        <div
          className="column is-half"
        >
          <figure
            className="center-image is-flex is-4 image is-256x256"
          >
            <img
              className="is-rounded"
              src="/imgs/SHPE_Icon.png"
            />
          </figure>
          <p
            className="title is-1 is-spaced has-text-centered mt-4"
          >
            Login
          </p>
          <div
            className="field"
          >
            <Component
              onClick={[Function]}
            />
            <Component
              className="mt-4"
              onClick={[Function]}
            />
            <Component
              className="mt-4"
              onClick={[Function]}
            />
          </div>
        </div>
      </div>
    </section>
  </div>
</Layout>
`;
